src/ScrumMaster/Slack/SlackNotifierResult.php

```diff
@@ -4,51 +4,58 @@
 
 namespace App\ScrumMaster\Slack;
 
+use App\ScrumMaster\Slack\ReadModel\SlackTicket;
+
 final class SlackNotifierResult
 {
-    /** @var array<string,int> */
-    private $codesPerTickets = [];
+    /** @var SlackTicket[] */
+    private $slackTickets = [];
 
-    public function addTicketKeyWithResponseCode(string $ticketKey, int $statusCode): void
+    public function addSlackTicket(SlackTicket $slackTicket): void
     {
-        $this->codesPerTickets[$ticketKey] = $statusCode;
+        $this->slackTickets[] = $slackTicket;
     }
 
-    /** @return array<string,int> */
+    /** @return SlackTicket[] */
     public function responseCodePerTickets(): array
     {
-        return $this->codesPerTickets;
+        return $this->slackTickets;
     }
 
     /** @return string[] */
     public function ticketKeys(): array
     {
-        return array_keys($this->codesPerTickets);
+        $keys = [];
+        foreach ($this->slackTickets as $slackTicket) {
+            $keys[] = $slackTicket->ticketCode();
+        }
+
+        return $keys;
```

Comment:
I think this is not really necessary if we keep the ticketKey as keys for `$his->slackTickets`. Don't you think so?

Reply:
:octocat: :+1: